assignment3.txt

```diff
@@ -0,0 +1,9 @@
+Julia Tan
+Assignment3 Part 2
+
+1. Yes, however it must be assumed that the variable 'i' has been initialized already.
```

Comment:
it's true that i needs to be initialized - this number also needs to be greater than 0 to not be caught in an infinite loop. Look at the print statement though - it's not properly written, so it would cause a compilation error.